### ISSUE TITLE
[feature] integration with other music services

### ISSUE THREAD

**Issue by olehmelnyk** [Stealth2012/meridian#43]
Any chance to add integration with other services, such as Google Music / Spotify / Pandora / Shazam / iTunes / etc.

**Comment by DeadMate** [comment]
Вот это хорошее предложение. Не прекращать же разработку из-за того, что владельцы VK жадины. Есть масса площадок типа prostoplayer или вышеупомянутых, с которыми можно замутить интеграции. + еще Soundcloud, а там вообще поле непаханное. Девелоперы, не вешайте нос!

**Comment by olehmelnyk** [comment]
да, я вот зарегистрировался на Spitify и Pandora через американский VPN - там дают слушать музыку бесплатно (правда с рекламой, которая частично "излечивается" из Adguard)

но напряжно регистрируясь на различных сервисах постоянно вручную добавлять любимые группы повсюду - вот если бы была интеграция чтоб можно было добавить любимые группы раз и список "любимого" обновлялся на всех площадках... + как-то подтягивать зашазамленые треки / исполнителей + список похожей музыки по версии различных сервисов
